pages/validator/external-chains/optimism.md

```diff
@@ -0,0 +1,440 @@
+   ⠦ 323993d60cf3 Waiting                                                                                                                                   91.6s
+   ⠦ 5149b9087ec2 Waiting                                                                                                                                   91.6s                                                                                                                             91.6s
+   ⠦ 209878a685e3 Waiting                                                                                                                                   91.6s
+ ⠼ fault-detector Pulling                                                                                                                                   97.4s
+```
+
+After it is done, verify by listing the services and their status
+```bash
+docker compose ps
+```
+
+<Tabs tabs={[
+{
+title: "Mainnet",
+content: <CodeBlock language="bash">
+{`
+#  You should see these 7 services running
+simple-optimism-node-dtl-1              "/bin/sh -c /scripts…"   dtl                 running             0.0.0.0:7878->7878/tcp, :::7878->7878/tcp
+simple-optimism-node-fault-detector-1   "docker-entrypoint.s…"   fault-detector      running             0.0.0.0:7301->7300/tcp, :::7301->7300/tcp
+simple-optimism-node-grafana-1          "/run.sh"                grafana             running             0.0.0.0:3000->3000/tcp, :::3000->3000/tcp
+simple-optimism-node-healthcheck-1      "npm run start"          healthcheck         running             0.0.0.0:7300->7300/tcp, :::7300->7300/tcp
+simple-optimism-node-influxdb-1         "/entrypoint.sh infl…"   influxdb            running             0.0.0.0:8086->8086/tcp, :::8086->8086/tcp
+simple-optimism-node-l2geth-1           "/bin/sh -c '/script…"   l2geth              running             0.0.0.0:9991-9992->8545-8546/tcp, :::9991-9992->8545-8546/tcp
+simple-optimism-node-prometheus-1       "/bin/prometheus --c…"   prometheus          running             0.0.0.0:9090->9090/tcp, :::9090->9090/tcp
+`}
+</CodeBlock>
+},
+{
+title: "Testnet",
+content: <CodeBlock language="bash">
+{`
+# You should see these 4 services running
+NAME                              COMMAND                  SERVICE             STATUS              PORTS
+simple-optimism-node-influxdb-1   "/entrypoint.sh infl…"   influxdb            running             0.0.0.0:8086->8086/tcp, :::8086->8086/tcp
+simple-optimism-node-op-geth-1    "/bin/sh -c '/script…"   op-geth             running             0.0.0.0:9991-9992->8545-8546/tcp, :::9991-9992->8545-8546/tcp
+simple-optimism-node-op-node-1    "/bin/sh -c '/script…"   op-node             running             
+simple-optimism-node-torrent-1    "/init"                  torrent             running             0.0.0.0:6881->6881/tcp, 0.0.0.0:6881->6881/udp, :::6881->6881/tcp, :::6881->6881/udp
+`}
+</CodeBlock>
+}
+]} />
+
+
+## Check logs
+
+## Testnet
```

Comment:
Should this be ###?

Reply:
Actually here I wanted to discuss something, I didn't want to make it separate headings at all, I want one headings with  Logs and then two tabs of tetsnet and mainnet, but I can't figure how to do that with the framework, the only time we have used tabs is with code blocks but if you use subheadings inside tabs, they start throwing error, I think we should support subheadings or text in tabs as well or is there any other way around it ?

Reply:
instead of including the subheadings inside the codeblock, you can isolate the differing content in different Tabs. If a user selects Testnet for a tab, all tabs on that page switch to Testnet, so I think that should be sufficient?

Reply:
Hmm, but the issue is number of service in mainnet is different from number of service in tetsnet (due to bedrock upgrade in testnet) - so we'll have uneven code blocks, meaning we will have something to write in mainnet block but nothing in tetsnet block, but let try it to see how it renders it.

Also if we have some text which is sepecfic to mainnet, I don't think the framework allows us to write some text and a code block in one tab

Reply:
Not an issue now because of https://github.com/axelarnetwork/axelar-docs/pull/306#discussion_r1086237875